File: 백준/Bronze/31746. SciComLove （2024）/SciComLove （2024）.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        
        String name = "SciComLove";
        int N = scan.nextInt();
        
        for(int i=0; i<N; i++) {
            String n = "";
        	for(int j=name.length(); j>0; j--) {
        		n += name.substring(j-1, j);
        	}
        	name = n;
        }
        System.out.println(name);
	}
}

File: 백준/Bronze/31746. SciComLove （2024）/SciComLove （2024）.py
A = "SciComLove"
N = int(input())

if(N%2 == 0) :
    print(A)
else :
    for i in range(2):
        n = ""
        for j in range(len(A), 0, -1):
            n += A[j-1:j]
        N = n
    print(N)